utils/media_info.js

```diff
@@ -3,21 +3,20 @@ const execFile = require('child_process').execFile;
 const fixPathname = require('../utils/fix_pathname');
 
 function parse(url) {
-	url = fixPathname(url);
 	return new Promise(function (resolve, reject) {
 		execFile('ffprobe',
```

Comment:
Please be aware of your function arguments and inline array indentation. You can have your reference from `lib/mp3_stream.js`